### ISSUE TITLE
WebGL is not working

### ISSUE THREAD

**Issue by HADB** [NeutroniumCore/Neutronium#103]
WebGL is not working in v1.5.0

**Comment by David-Desmaisons** [comment]
Could you create a project with a minimal set-up with this problem, so I can investigate?
Thanks.

**Comment by HADB** [comment]
OK, I will give you next week.
PS: I'm trying to show a web page in iframe which using WebGL and tells me WebGL not working.

**Comment by David-Desmaisons** [comment]
Meanwhile try to check the log and verify if you need to set a  [CEFswitch ](https://neutroniumcore.github.io/Neutronium/articles/chromium-api.html)

**Comment by HADB** [comment]
Hi, I have set up a minimal project https://github.com/HADB/Neutronium.Vue.SPA.WebGL

What I have done:
1. Create a project with Neutronium.Vue.SPA template
2. Edit View/src/main.vue, add an iframe element which contains a WebGL website source
```
<template>
    <main>
      <v-content>
        <v-container fluid class="main-container">
          <iframe src="https://webglsamples.org/aquarium/aquarium.html" />
        </v-container>
      </v-content>
    </main>
</template>

<style>
...
iframe{
  width: 100%;
  height: 100%;
}
...
</style>
```
3. Build and run

Screenshots:
1. The WebGL demo in chrome
![chrome](https://user-images.githubusercontent.com/1836701/52104336-0de0ea00-2625-11e9-8661-a830c3e2736e.PNG)
2. Result in Neutronium app
![demo](https://user-images.githubusercontent.com/1836701/52104384-61ebce80-2625-11e9-8cd9-005b8dbc6921.png)

**Comment by HADB** [comment]
Maybe this is related to the following line but I didn't test because I couldn't find a way to remove this CEFswitch.

https://github.com/NeutroniumCore/Neutronium/blob/8603928f9d8101e48e7621917212119b0404c212/WebBrowserEngine/ChromiumFX/HTMEngine.ChromiumFX/ChromiumFxWebBrowserApp.cs#L23

**Comment by David-Desmaisons** [comment]
@HADB I will provide in version 1.5.1 a way to not disable Gpu. On ChromiumFxWebBrowserApp:
```CSharp
        /// <summary>
        /// Set to true to allow Gpu
        /// </summary>
        protected virtual bool AllowGpu => false;
```

I remember that the past using GPU could introduce some bugs in display so by default Gpu will continue to be disabled.

**Comment by David-Desmaisons** [comment]
Available in version 1.5.2

**Comment by HADB** [comment]
Problem solved. Thanks!
